solutions/main.tf

```diff
@@ -1,7 +1,7 @@
 provider "aws" {
-  region = "ap-southeast-1"
-  access_key = "ACCESS_KEY"
-  secret_key = "SECRET_KEY"
+  region = "ap-southeast-1a"
+  access_key = "AKIAJ4LH6ACCQS6OKZ5A"
+  secret_key = "BQ03xxQNkI3/I9xs4F5U5RsZCXU7LNlmyGetfkDM"
```

Comment:
It's dummy key. AWS rule state that never commit key to github.
From my opinion, put the key when you need to create new instance